internals/secrethub/run.go

```diff
@@ -87,6 +87,13 @@ func (cmd *RunCommand) Run() error {
 	}
 	envSources = append(envSources, flagSource)
 
+	if cmd.template == "" {
+		const defaultTemplate = "secrethub.env"
+		if _, err := os.Stat(defaultTemplate); err == nil {
```

Comment:
Probably safer like this? Because if it does exist and there is an error, you do want to know about it.
```go
if _, err := os.Stat(defaultTemplate); os.IsNotExist(err) {
   cmd.template = defaultTemplate
}
```